app/models/team.rb

```diff
@@ -12,8 +12,21 @@ class Team < ActiveRecord::Base
 
   # == Instance methods
   def score
-    total_score = users.map{ |u| u.score }.sum
+    bills = self.users.map { |user| user.most_recent_bills }
+    bills.reject { |score| !score.kind_of?(Array) | (score.length < 2) }
```

Comment:
I can probably tighten this up by having this logic go into most_recent_bills in user.rb and having it return a nil if it lacks two bills.